code/game/machinery/adv_med.dm

```diff
@@ -118,7 +118,7 @@
 				var/obj/structure/LB = L.buckled
 				LB.user_unbuckle_mob(user)
 			var/mob/M = G.affecting
-			if (M.client)
+			if (M && istype(M) && M.client)
```

Comment:
This can be simplified to `istype(M) && M.client`, `null` is always false in istype.